iOS/UI/Library/LibraryViewController.swift

```diff
@@ -804,7 +804,7 @@ extension LibraryViewController {
             }
         }
         let trackingFilter: [UIAction]
-        if TrackerManager.shared.hasAvailableTrackers {
+        if TrackerManager.shared.hasAvailableTrackers(manga: nil) {
```

Comment:
shouldn't you provide the manga here? i'm a little confused why it's used in some places and not others